apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -1,13 +1,14 @@
 import React, {PropTypes} from 'react';
-import {personalProjectDataPropType} from './projectConstants';
 import PersonalRecentProjects from './PersonalRecentProjects.jsx';
 import NewProjectButtons from './NewProjectButtons.jsx';
 import ContentContainer from '../ContentContainer.jsx';
 import i18n from "@cdo/locale";
+import _ from 'lodash';
 
 const ProjectWidget = React.createClass({
   propTypes: {
-    projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
+    projectList: PropTypes.array,
```

Comment:
nit: I'd recommend making projectList required here, otherwise you risk passing `undefined` which might break `convertChannelsToProjectData`